forwards_paper.tex

```diff
@@ -335,17 +330,97 @@ \subsection*{Recording the pedigree in forwards time}
 from $x$ to $L$ is $P_b$. We then complete the replacement event by incrementing
 $n$, ready to represent the next new node.
 
+This algorithm records only the topological information resulting from the
+forwards in time Wright-Fisher process, but it is straightforward to add
+mutational information. This can be done in two different ways.
```

Comment:
should clarify that if mutations are selected then we'd need to do it on the fly.

Reply:
That's true. I'm not entirely happy with the flow of thought through this section. It'd probably be good to talk it through and see if we can come up with a clear narrative.